website/files/models/base.py

```diff
@@ -737,7 +738,7 @@ def update_metadata(self, metadata, save=True):
         # If its are not in this callback it'll be in the next
         self.size = self.metadata.get('size', self.size)
         self.content_type = self.metadata.get('contentType', self.content_type)
-        if self.metadata.get('modified') is not None:
+        if self.metadata.get('modified') is not None and self.metadata.get('modified') != '':
```

Comment:
Minor: it's safer and more concise to check the truthiness of `self.metadata.get('modified')`. I can make this change when I merge.